### PR TITLE
Update to Test Plan documentation 

### PR DESCRIPTION
**Pull request by Prome88**
References https://stackoverflow.com/questions/59747294/missing-default-components-tab-in-test-plan

### CODE REVIEW

**Comment by Prome88** [comment]
Will fix doc8 reported issues and update the commit:

- docs/source/guide/testplan.rst:34: D000 malformed hyperlink target.
- docs/source/guide/testplan.rst:92: D000 Bullet list ends without a blank line; unexpected unindent.
- docs/source/guide/testplan.rst:95: D000 Bullet list ends without a blank line; unexpected unindent.
- docs/source/guide/testplan.rst:93: D002 Trailing whitespace
- docs/source/guide/testplan.rst:96: D002 Trailing whitespace
- docs/source/guide/testplan.rst:121: D002 Trailing whitespace
- docs/source/guide/testplan.rst:222: D002 Trailing whitespace
- docs/source/guide/testplan.rst:224: D002 Trailing whitespace
- docs/source/guide/testplan.rst:229: D002 Trailing whitespace

File: docs/source/guide/testplan.rst [hunk 1]
```diff
@@ -31,7 +31,7 @@ Other Test Plans can be searched via the **SEARCH** item in the **Main menu**:
    |Click column to sort by.|
 
 
-.. note::
+.. _note:
```
Comment (atodorov) [inline docs/source/guide/testplan.rst]: `.. note:` is a Sphinx directive. `.. _cloning-testplan:` is an anchor so we can link to this section of the document from elsewhere.

Revert this change.

File: docs/source/guide/testplan.rst [hunk 2]
```diff
@@ -91,10 +85,14 @@ product or a new version of the current product. To clone a Test Plan:
...
 #. In the **Clone Test Plan** screen, perform the following actions:
 
-   -  Enter the **New Plan Name**.
-   -  In **Copy to Product**, select the product.
-   -  In **Product Version**, select the version.
-   -  Tick the **Copy Settings** you want
+   -  Enter the **Name**.
+   -  In **Product**, select the product.
+   -  In **Version**, select the version.
+   -  Tick the **Clone TCs** you want to clone test cases, otherwise
```
Comment (atodorov) [inline docs/source/guide/testplan.rst]: `if you want`
^^ missing

File: docs/source/guide/testplan.rst [hunk 3]
```diff
@@ -282,19 +257,17 @@ Kiwi TCMS offers functionality to export test cases via `TCMS API <https://kiwit
 .. |The Search menu| image:: ../_static/Click_Search.png
 .. |The Search Plan screen| image:: ../_static/Planning_Home.png
 .. |Click column to sort by.| image:: ../_static/Sort_By_Column.png
-.. |The Clone plan button| image:: ../_static/Plan_Actions.png
+.. |The Clone plan button| image:: ../_static/Plan_Actions_Clone.png
 .. |The Clone Test Plan screen| image:: ../_static/Clone_Details.png
-.. |The Edit plan button| image:: ../_static/Plan_Actions.png
+.. |The Edit plan button| image:: ../_static/Plan_Actions_Edit.png
```
Comment (atodorov) [inline docs/source/guide/testplan.rst]: These buttons do not get highlighted. I believe the image is incorrect. 

There is no need to specific images for every action button. 1 image (like it was before) without the highlight should be enough. Then you don't have to change the name of this file.

File: docs/source/guide/testplan.rst (hunk 1 above)
Comment (Prome88) [reply docs/source/guide/testplan.rst]: Then we need to disable this check in doc8 as it complained about it.

Comment (atodorov) [reply docs/source/guide/testplan.rst]: Are you sure you are using the latest & greatest of versions ? I don't see any complains from doc8 on the master branch which is without your change https://travis-ci.org/kiwitcms/Kiwi/jobs/637678567. 

Make sure to revert this lline.

Comment (Prome88) [reply docs/source/guide/testplan.rst]: Done.

File: docs/source/guide/testplan.rst (hunk 2 above)
Comment (Prome88) [reply docs/source/guide/testplan.rst]: Added.

File: docs/source/guide/testplan.rst (hunk 3 above)
Comment (Prome88) [reply docs/source/guide/testplan.rst]: One image, without highlighted button.

File: docs/source/guide/testplan.rst [hunk 4]
```diff
@@ -31,7 +31,7 @@ Other Test Plans can be searched via the **SEARCH** item in the **Main menu**:
    |Click column to sort by.|
 
 
-.. note::
+.. note:
```
Comment (atodorov) [inline docs/source/guide/testplan.rst]: Revert this change

File: docs/source/guide/testplan.rst [hunk 5]
```diff
@@ -87,14 +81,18 @@ product or a new version of the current product. To clone a Test Plan:
...
-   -  Enter the **New Plan Name**.
-   -  In **Copy to Product**, select the product.
-   -  In **Product Version**, select the version.
-   -  Tick the **Copy Settings** you want
+   -  Enter the **Name**.
+   -  In **Product**, select the product.
+   -  In **Version**, select the version.
+   -  Tick the **Clone TCs** if you want to clone test cases, otherwise
+      they will be only linked into new TP
+
```
Comment (atodorov) [inline docs/source/guide/testplan.rst]: Remove empty line
